### PR TITLE
Add packages for all platforms and script to update the packages

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Review by chee — APPROVED** [review]
yay, good work

File: index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        return require("@financial-times/sass-linux-x64");
+      } catch {
+        try {
+          return require("@financial-times/sass-linux-ia32");
+        } catch {
+          try {
+            return require("@financial-times/sass-macos/x64");
+          } catch {
+            throw new Error(
+              "oax does not have a precompiled binary for the platform/architecture you are using. Please contact Origami or open an issue on https://github.com/Financial-Times/oax/issues"
```
Comment (JakeChampion) [inline index.js]: oax....
```suggestion
              "@financial-time/sass does not have a precompiled binary for the platform/architecture you are using. Please contact Origami or open an issue on https://github.com/Financial-Times/sass/issues"
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      return require("@financial-times/sass-windows-ia32");
+    } catch {
+      try {
+        return require("@financial-times/sass-linux-x64");
+      } catch {
+        try {
+          return require("@financial-times/sass-linux-ia32");
+        } catch {
+          try {
+            return require("@financial-times/sass-macos/x64");
```
Comment (JakeChampion) [inline index.js]:
```suggestion
            return require("@financial-times/sass-macos-x64");
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@financial-times/sass",
+  "version": "1.26.2",
```
Comment (JakeChampion) [inline package.json:3]: This tracks the version of dart-sass

File: sass-linux-ia32/package.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@financial-times/sass-macos-ia32",
```
Comment (JakeChampion) [inline sass-linux-ia32/package.json]:
```suggestion
  "name": "@financial-times/sass-linux-ia32",
```

File: sass-linux-x64/package.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@financial-times/sass-macos-x64",
```
Comment (JakeChampion) [inline sass-linux-x64/package.json]:
```suggestion
  "name": "@financial-times/sass-linux-x64",
```

File: package.json (hunk 3 above)
Comment (chee) [reply package.json:3]: is this a good idea? what if we need to release a patch of our own to `@financial-times/sass`?

Comment (JakeChampion) [reply package.json:3]: I've changed it to use whatever version we choose

**Pull request merged**